docs/guides/admin/docs/configuration/log.md

```diff
@@ -0,0 +1,60 @@
+Log
+===
+
+The settings for logging can be found in:
+
+    .../etc/org.ops4j.pax.logging.cfg
+
+Each Log4J appender can be configured in a similar fashion to the graylog example down below. The following
+requirements have to be met:
+* It needs to be a Log4J appender
+* The used bundle needs to be a fragment-bundle
+
+Graylog
+-------
+
+To have all log data available and accessible in one central location one can use graylog. A guide to install
+graylog can be found [here](http://docs.graylog.org/en/stable/).
+
+
+Add gelfj-X.X.X.jar (works up to version 1.1.14) to the appropriate folder in the karaf system folder (e.g. `/system/org/graylog2/gelfj/X.X.X/gelfj-X.X.X.jar`)
+The directory has the same structure as a maven repository!

```

Comment:
If this will get into the Opencast Codebase. I think it schould be integrated during the build process. So as a dependency in the main pom where the ops4j is included. @lkiesow whats your opinion about this?

Reply:
getting it into Opencast codebase would have been our second step, if people are satisfied and interested in "more".

question is: does it work for you after all?

Reply:
@ziegenberg  sorry for the late reply. I did not get it running on the opencast side...

Reply:
@ziegenberg could you send me a step by step documentation, with full example, how you did this?

Reply:
@CGreweling do you get an error message from karaf or just no logs in graylog?

Reply:
@geichelberger: i downloaded a precompiled jar, which didn't work. maven build did not work from source. But i got it running now with version 1.1.13 and gradle build.

Reply:
@ziegenberg  sorry, for the long delay. If you could add an example and resolve the conflicts i will merge this.